packages/sources/react/src/components/VtmnTextInput/VtmnTextInput.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import clsx from 'clsx';
+
+import '@vtmn/css-text-input';
+
+export interface VtmnTextInputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
```

Comment:
Could you please follow the same method that we use with `VtmnButtonProps` proposed by @FredericEspiau, please?

```suggestion
export interface VtmnTextInputProps
  extends React.ComponentPropsWithoutRef<'input'> {
```